game.py

```diff
@@ -1,99 +1,239 @@
 import sys
 import time
 import tkinter as tk
+import pygame 
 from random import choice, randint, shuffle
 from tkinter import messagebox
+from ttkbootstrap.style import Style 
 
+# Importações do ttkbootstrap
+import ttkbootstrap as ttk
+from ttkbootstrap.dialogs import Messagebox
 
 class ChessChallenge:
+    
     RULES = {
-        "FULL_DIAGONAL" : lambda qr, qc, r, c: abs(qr - r) == abs(qc - c),
-        "ADJACENT_DIAGONAL" : lambda qr, qc, r, c: qr - 1 <= r <= qr + 1 and qc - 1 <= c <= qc + 1
+        "FULL_DIAGONAL": lambda qr, qc, r, c: abs(qr - r) == abs(qc - c),
+        "ADJACENT_DIAGONAL": lambda qr, qc, r, c: qr - 1 <= r <= qr + 1 and qc - 1 <= c <= qc + 1
     }
 
     def __init__(self, root, size):
-        self.size = size  # Tamanho do tabuleiro
+        """Inicializa o jogo, a interface gráfica e define as regras."""
+        self.size = size  # Tamanho do tabuleiro (N)
         self.root = root
         self.root.title(ChessChallenge.title())
-        #FIXME put the window on center of screen
-        # screen_width = self.root.winfo_screenwidth()
-        # screen_height = self.root.winfo_screenheight()
-        window_width = 50 * size
-        window_height = 50 * size
-        # pos_x = (screen_width // 2 - window_width) // 2
-        # pos_y = (screen_height - window_height) // 3
-        # self.root.geometry(f"{window_width}x{window_height}+{pos_x}+{pos_y}")
+
+        # Configuração para centralizar a janela na tela
+        screen_width = self.root.winfo_screenwidth()
+        screen_height = self.root.winfo_screenheight()
+        window_width = 100 * size
+        window_height = 70 * size
+        pos_x = (screen_width - window_width) // 2
+        pos_y = (screen_height - window_height) // 2 
+        
+        self.root.geometry(f"{window_width}x{window_height}+{pos_x}+{pos_y}")
         self.root.protocol("WM_DELETE_WINDOW", self.close_check_data)
+        self.root.minsize(window_width, window_height)
+        
         self.canvas = tk.Canvas(root, width=window_width, height=window_height)
-        self.canvas.pack()
-        response = messagebox.askyesno("Regras!", "Usar a regra de diagonal adjacente?")
-        if response:
-            self.diagonal_rule = self.RULES.get("ADJACENT_DIAGONAL")
-        else:
-            self.diagonal_rule = self.RULES.get("FULL_DIAGONAL")
-        self.solution = self.colors = self.queens = []
-        self.board = [[None for _ in range(size)] for _ in range(size)]
-        self.start_time = None
+        self.menu_frame = ttk.Frame(root) 
+
+        # self.MUSIC_FILES = {
+        #     "menu": "music/BalatroMainTheme.mp3", 
+        #     "game": "music/BalatroBossTheme.mp3" 
+        # }
+        # self.current_music = None
+
+        # try:
+        #     pygame.mixer.init()
+        # except pygame.error as e:
+        #     print("Erro ao tocar a música: {e}. Música off")
+        #     self.MUSIC_FILES = {}
```

Comment:
Maybe create an empty mp3 folder, so the player can put the mp3 they want to listen to.

Reply:
only create an empty folder X, Only plays if have files there.